plotly/offline/offline.py

```diff
@@ -8,40 +8,29 @@
 import json
 import os
 import uuid
-
-import requests
+from pkg_resources import resource_string
+import warnings
 
 from plotly import session, tools, utils
 from plotly.exceptions import PlotlyError
 
-PLOTLY_OFFLINE_DIRECTORY = plotlyjs_path = os.path.expanduser(
-    os.path.join(*'~/.plotly/plotlyjs'.split('/')))
-PLOTLY_OFFLINE_BUNDLE = os.path.join(PLOTLY_OFFLINE_DIRECTORY,
-                                     'plotly-ipython-offline-bundle.js')
-
 
 __PLOTLY_OFFLINE_INITIALIZED = False
 
 
 def download_plotlyjs(download_url):
-    if not os.path.exists(PLOTLY_OFFLINE_DIRECTORY):
-        os.makedirs(PLOTLY_OFFLINE_DIRECTORY)
-
-    res = requests.get(download_url)
-    res.raise_for_status()
+    warnings.warn('''
+        `download_plotlyjs` is deprecated and will be removed in the
+        next release. plotly.js is shipped with this module, it is no
+        longer necessary to download this bundle separately.
+    ''', DeprecationWarning)
+    pass
 
-    with open(PLOTLY_OFFLINE_BUNDLE, 'wb') as f:
-        f.write(res.content)
 
-    print('\n'.join([
-        'Success! Now start an IPython notebook and run the following ' +
-        'code to make your first offline graph:',
-        '',
-        'import plotly',
-        'plotly.offline.init_notebook_mode() '
-        '# run at the start of every ipython notebook',
-        'plotly.offline.iplot([{"x": [1, 2, 3], "y": [3, 1, 6]}])'
-    ]))
+def get_plotlyjs():
+    path = os.path.join('offline', 'plotly.min.js')
+    plotlyjs = resource_string('plotly', path).decode('utf-8')
```

Comment:
I'm a little confused how this is working? I thought you needed a line in `setup.py` telling the setup script to include the `.js` file in the package information. Like [here](https://github.com/plotly/plotly.py/blob/master/setup.py#L43).

I think you need a `'offline/*.js'` item in that list?